src/kili/mutations/asset/__init__.py

```diff
@@ -342,9 +348,9 @@ def generate_variables(batch):
             return {"where": {"idIn": batch["asset_ids"]}}
 
         @retry(
-            stop=stop_after_delay(5),
             wait=wait_fixed(1),
```

Comment:
I think that we should check with @utix (aurélien) and @HNicolas to validate such a decision.

Reply:
I agree with you indeed. We can use an exponential wait, we start at 1 sec and double the delay at every retry with a max wait of 16s. Because I don't think we can do better SDK side for now

Reply:
Also, can't we check the last asset of the last batch only? Instead of checking the whole batch? I'm not sure if it's processed in parallel in the backend or not

Reply:
Ah yeah, good point, I don't know how it is handled exactly, we could have a look at the label-import service or the appendManyFramesToDatasetAsynchronously.
But also, it is maybe possible in some cases to have asynchronous batches between synchronous ones. If we test the last batch, we wouldn't catch the asynchronous behavior of the in-between batches. Might be worth to check if all asynchronous batches are done last in the SDK ?

Reply:
I added indefinite retries with exponential delay (min=1, max=8) and query only last element of last batch of the mutation